robot_localization/pf.py

```diff
@@ -48,6 +50,21 @@ def as_pose(self):
                     orientation=Quaternion(x=q[0], y=q[1], z=q[2], w=q[3]))
 
     # TODO: define additional helper functions if needed
+    def update_pose(self, delta_pos_particle, delta_ang):
+
```

Comment:
docstring stating in particular format/datatype of delta_pos_particle would be helpful but comments make it non critical, good comments!

Reply:
Docstring added with summary, args, returns, and variable types.